### ISSUE TITLE
cmd+a on macos azerty keyboard quits the app

### ISSUE THREAD

**Issue by kud** [Kilian/fromscratch#95]
Hello!

I don't know if it's only for azerty keyboard or any keyboard but cmd+a quits the app instead of selecting the whole text.

**Comment by Kilian** [comment]
Yes, the shortcuts map to physical locations, so cmd+a on an azerty keyboard registers as cmd+q, which is... quit.

This is an electron bug unfortunately. There are ways around it but I haven't had the time to fix them yet. 

If you're happy to build/run your own, then you can clone this repository and remove these lines:
https://github.com/Kilian/fromscratch/blob/master/app/main.dev.js#L193-L195
That'll fix the issue.

**Comment by kud** [comment]
Okay thank you for your feedback.

To be honest, I wanted to compile it and try to remove this line but I wasn't able to compile the project. I've got some errors.

So I'll ask directly to you this:

I would like to know why do you bind this "quit" command, as cmd+q is the command by default to quit any app?

Maybe we could remove this line and let electron manages it by itself to quit?

**Comment by kud** [comment]
-> https://github.com/Kilian/fromscratch/pull/101 

does it work for you?

**Comment by Kilian** [comment]
That's not how it works. No app will "let the OS handle it". The line I linked can be removed because the following menu item already has an accelerator linked:

https://github.com/Kilian/fromscratch/blob/master/app/main.dev.js#L280-L284

Menu accelerators do not have the issue that global shortcuts have and will correctly translate between keyboard layouts. So the global shortcut is redundant, and I didn't know that when I originally created this application. A future release will remove all redundant shortcuts.

**Comment by kud** [comment]
Thank you 😊
